docs/how-to/reload-loader-data-after-command-dispatcher.mdx

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 1
+title: Reload loader data after command dispatcher
+description:
+  In V3 we mix GraphQL calls made in remix data loader and client-side requests
+  with Apollo. Here you will learn how to reload loader data after command
+  execution
+---
+
+:::caution
+
+This part of the documentation is still a work in progress. This reflect the
+actual state of the work done in V3.
+
+Please consider it in an unstable state and it may be subject to changes.
+
+:::
+
+<p>{frontMatter.description}</p>
+
+# Philosophy
+
+In V3, we decided to use
+[Remix data loader](https://remix.run/docs/en/main/route/loader) to provide the
+fastest SSR rendering possible.
+
+But some client side actions (like update a customer address) are done client
+side for better visitor experience using `makeCommandDispatcher`.
```

Comment:
I don't think this statement is correct, this is a compat thing, and in the future, we will aim to do everything through actions. So it's not for better visitor experience.

Reply:
I've updated the section, looking forward to your suggestions :)